doc/man1/openssl.pod

```diff
@@ -781,6 +781,257 @@ client.
 The input format for the extra certificate and key, respectively.
 See L<openssl(1)/Format Options> for details.
 
+=item B<-xchain_build>
+
+Specify whether the application should build the certificate chain to be
+provided to the server for the extra certificates via the B<-xkey>,
+B<-xcert>, and B<-xchain> options.
+
+=item B<-xcertform> B<DER>|B<PEM>, B<-xkeyform> B<DER>|B<PEM>
+
+The input format for the extra certifcate and key, respectively.
+See L<openssl(1)/Format Options> for details.
+
+=back
+
+=head2 Verification Options
+
+Many OpenSSL commands verify certificates. The details of how each
+command handles errors is documented on the specific command page.
```

Comment:
Nit: are documented

Reply:
Nope, "each command" is singular, short for "each one of the commands" :)

Reply:
But it's the *details* (plural) that are being documented here.  Er, there.  Whatever.

Reply:
I agree with @beldmit and @kaduk. So that's 3:1. I suggest that I'll just fix it while merging.